cron/internal/worker/main.go

```diff
@@ -91,7 +118,11 @@ func newScorecardWorker() (*ScorecardWorker, error) {
 
 	sw.ctx = context.Background()
 	sw.logger = log.NewLogger(log.InfoLevel)
-	sw.repoClient = githubrepo.CreateGithubRepoClient(sw.ctx, sw.logger)
+	sw.githubClient = githubrepo.CreateGithubRepoClient(sw.ctx, sw.logger)
+	// TODO(raghavkaul): Read GitLab auth token from environment
+	if sw.gitlabClient, err = gitlabrepo.CreateGitlabClient(sw.ctx, "gitlab.com"); err != nil {
```

Comment:
the baseurls are expected to include the scheme, or you'll get repo unreachable errors for every repo

can we make this a constant with value `https://gitlab.com/`?